src/api/affiliate.py

```diff
@@ -0,0 +1,35 @@
+"""Basic Flask API"""
+import os
+from typing import Any
+
+from dotenv import load_dotenv
+from dune_client.client import DuneClient
+from flask import Flask, jsonify
+
+from src.fetch.affiliate_data import CachingAffiliateFetcher
+from src.logger import set_log
+
+app = Flask(__name__)
+load_dotenv()
+cached_fetcher = CachingAffiliateFetcher(
+    dune=DuneClient(os.environ["DUNE_API_KEY"]),
+    execution_id=os.environ.get("LAST_EXECUTION_ID"),
+    cache_validity=int(os.environ["CACHE_VALIDITY"]),
+)
+log = set_log(__name__)
+
+
+@app.route("/profile/<string:address>", methods=["GET"])
+def get_profile(address: str) -> Any:
+    """
+    Fetches Affiliate Profile for `address`
+    https://api.cow.fi/affiliate/api/v1/profile/<address>
+    """
+    log.info(f"Fetching affiliate data for {address}")
+    affiliate_data = cached_fetcher.get_affiliate_data(account=address)
+    log.debug(f"got results {affiliate_data}")
```

Comment:
Aren't you double logging this info (once here and once in `get_affiliate_data`)? Let's make sure we log as much as necessary (to debug things) but as little as possible and also follow proper log line format.

Also, aren't we supposed to use app.logger (https://flask.palletsprojects.com/en/1.1.x/logging/)?

Reply:
I wasn't aware of app logger but will definitely check it out. Hadn't noticed the double logs, but will remove.